libusb.go

```diff
@@ -217,6 +219,24 @@ func (libusbImpl) getDevices(ctx *libusbContext) ([]*libusbDevice, error) {
 	return ret, nil
 }
 
+func (libusbImpl) wrapSysDevice(ctx *libusbContext, systemDeviceHandle int) (*libusbDevHandle, error) {
+	var handle *C.libusb_device_handle
+	if ret := C.libusb_wrap_sys_device((*C.libusb_context)(ctx), C.intptr_t(systemDeviceHandle), &handle); ret < 0 {
+		return nil, fromErrNo(C.int(ret))
+	}
+
+	return (*libusbDevHandle)(handle), nil
+}
+
+func (libusbImpl) getDevice(d *libusbDevHandle) (*libusbDevice, error) {
```

Comment:
libusb_get_device doesn't return errors, so this should probably be just a trivial wrapper:

```
func (...) getDevice(*d libusbDevHandle) *libusbDevice {
  return C.lbusb_get_device(...)
}
```